python/ray/dataframe/test/test_dataframe.py

```diff
@@ -3104,32 +2984,32 @@ def test_unstack():
 
 
 def test_update():
-    df = rdf.DataFrame([[1.5, np.nan, 3.],
-                        [1.5, np.nan, 3.],
-                        [1.5, np.nan, 3],
-                        [1.5, np.nan, 3]])
+    df = pd.DataFrame([[1.5, np.nan, 3.],
```

Comment:
See above.

Reply:
I don't think changing this one is necessary. It's used once.